beacon_node/beacon_chain/src/validator_monitor.rs

```diff
@@ -553,17 +633,21 @@ impl<T: EthSpec> ValidatorMonitor<T> {
                             );
                         }
                     } else {
-                        metrics::set_int_gauge(
-                            &metrics::VALIDATOR_MONITOR_VALIDATOR_IN_CURRENT_SYNC_COMMITTEE,
-                            &[id],
-                            0,
-                        );
-                        debug!(
-                            self.log,
-                            "Validator isn't part of the current sync committee";
-                            "epoch" => current_epoch,
-                            "validator" => id,
-                        );
+                        if self.individual_tracking() {
+                            metrics::set_int_gauge(
+                                &metrics::VALIDATOR_MONITOR_VALIDATOR_IN_CURRENT_SYNC_COMMITTEE,
+                                &[id],
+                                0,
+                            );
+                        }
+                        if self.individual_tracking() {
+                            debug!(
+                                self.log,
+                                "Validator isn't part of the current sync committee";
+                                "epoch" => current_epoch,
+                                "validator" => id,
+                            );
+                        }
```

Comment:
We could collapse these into one `if`, or into the `else` of the parent expression

Reply:
Good suggestion, included in 6eb7dc249